README.md

```diff
@@ -0,0 +1,28 @@
+![](./images/Logo_.png)
+
+Cine Encoder for Linux is an application that allows you to convert media files while preserving HDR signals. NVENC hardware encoding for NVIDIA video cards is also supported for H265 and H264 modes. The following encoding modes are implemented:
```

Comment:
No one reads this file, this information could go to the appdata file if it's not there already.